examples/statistics/errorbar_limits.py

```diff
@@ -1,50 +1,76 @@
 """
-Demo of the errorbar function, including upper and lower limits
+===========================================================
+Demo of how to include upper and lower limits in error bars
+===========================================================
+
+In matplotlib, errors bars can have "limits". Applying limits to the
+error bars essentially makes the error unidirectional. Because of that,
+upper and lower limits can be applied in both the y- and x-directions
+via the `uplims`, `lolims`, `xuplims`, and `xlolims` parameters,
+respectively. This parameters can be scalar or boolean arrays.
+
+For example, if `xlolims` is `True`, the x-error bars will only extend
+from the data towards increasing values. If `uplims` is an array filled
+with `False` except for the 3rd and 5th values, all of the y-error bars
```

Comment:
I believe the example below uses the 4th and 7th values, so maybe better to use that here.